src/browser/templates/repo-browse-tables.html

```diff
@@ -28,6 +28,12 @@
             </td>
             
             <td class="text-right">
+
+              {% if login == repo_base %} 
+              <a href="/browse/{{repo_base}}/{{repo}}/securitypolicies/{{base_table}}" title="View Security Policies"><span class="fa fa-lock"></span></a>
```

Comment:
Use the url tag direct to a named url
https://docs.djangoproject.com/en/1.9/ref/templates/builtins/#url

You can see an example here:
https://github.com/datahuborg/datahub/blob/master/src/browser/templates/layout.html#L216